### PR TITLE
Balance Mini_geo

### PR DESCRIPTION
**Pull request by skymyj**

### CODE REVIEW

File: units/tllgeo_mini.lua [hunk 1]
```diff
@@ -26,10 +26,11 @@ return {
 		losemitheight = 29,
 		makesmetal = 3.5,
 		mass = 1710,
-		maxdamage = 950,
+		maxdamage = 800,
 		maxslope = 20,
 		maxwaterdepth = 0,
-		metalstorage = 100,
+		metalproduce = 3,
```
Comment (nixtux) [inline units/tllgeo_mini.lua]: It's makesmetal

**Comment by skymyj** [comment]
Oups. Sorry. 
Correction done.

**Comment by nixtux** [comment]
Should we make a morph to t1 ?

**Comment by skymyj** [comment]
Nope imho. 
Just be easy. Goal today is to delete units and to balance tll faction

**Pull request merged**